src/Particle.C

```diff
@@ -17,11 +17,10 @@ void Particle::pup(PUP::er &p) {
 }
 
 void Particle::reset() {
-  density = 0.0;
-  pressure = 0.0;
-  potential = 0.0;
-  acceleration = 0.0;
-  // TODO velocity?
+  density       = 0.0;
+  pressure      = 0.0;
+  potential     = 0.0;
+  acceleration  = Vector3D<Real> (0.0, 0.0, 0.0);
```

Comment:
Minor concern, but should whitespace changes be made into separate PR's? We could go through and reformat the entire codebase in a PR and to keep `git blame` consistent.

Reply:
thats fair I wont do it again. But I do want my name on this whole function because it is a significant change to it